### ISSUE TITLE
Adding .data and .grad support for syft tensors

### ISSUE THREAD

**Issue by LaRiffle** [OpenMined/PySyft#1895]
So far, interesting work has been done to support .data and .grad (attributes of parameters) with pointers. As we start using syft tensors such as fix_precision_tensor it is important to add a native behaviour to those to handle these attributes calls.
I would suggest doing imitating pointer on this. I'm open to discussion this in details.

**Comment by iamtrask** [comment]
I think we can add .grad and .data creation to __init__ of AbstractTensor which would make this work.

**Comment by LaRiffle** [comment]
Yes I agree!

**Comment by mcleonard** [comment]
Working on AutogradTensor and running into this issue. I'm storing gradients after backprop to `.grad` on Autograd tensors, but the gradients aren't available on the Wrapper at the top of the chain with `.grad`. I think we'll need `.grad` to work on Wrappers to use PyTorch's optimizers.

**Comment by gmuraru** [comment]
@LaRiffle, @karlhigley  we could close this issue, right?
